common/key/store_test.go

```diff
@@ -70,9 +67,26 @@ func TestKeysSaveLoad(t *testing.T) {
 		Scheme: group.Scheme,
 	}
 	require.Nil(t, store.SaveShare(testShare))
-	loadedShare, err := store.LoadShare(group.Scheme)
+	loadedShare, err := store.LoadShare()
 
 	require.NoError(t, err)
 	require.Equal(t, testShare.Share.V, loadedShare.Share.V)
 	require.Equal(t, testShare.Share.I, loadedShare.Share.I)
 }
+
+func TestTwoStores(t *testing.T) {
+	// we don't use the function from the test package here to avoid a circular dependency
```

Comment:
damn golang >.>

Reply:
That's weird I have 0 recollection of adding that test and having this problem xD